src/style.css

```diff
@@ -0,0 +1,148 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf) format("truetype");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: flex-start;
+  margin: 25px 0;
+  padding-left: 40px;
+  width: 100%;
+  margin-left: 20px;
+}
+
+.search-form__input {
+  flex: 1;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: "Avenir", sans-serif;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+  padding-left: 40px;
+  padding-right: 40px;
+  font-size: 16px;
+  line-height: 22px;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.search-form_big .search-form__input {
+  height: 70px;
+}
+
+.search-form_small .search-form__input {
+  height: 42px;
+}
+
+.search-form__container {
+  position: relative;
+  width: 1050px;
+}
+
+.search-form__label {
+  display: flex;
```

Comment:
You don't really need to use flex in this task. Will be better to use absolute position for the search image